android/src/main/java/com/razorpay/rn/RazorpayModule.java

```diff
@@ -16,6 +16,9 @@
 import com.facebook.react.bridge.Arguments;
 import com.facebook.react.modules.core.DeviceEventManagerModule;
 import com.razorpay.CheckoutActivity;
+import com.razorpay.PaymentData;
```

Comment:
We've already exposed this for the auto-capture flow, no harm in exposing it here